components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -206,6 +207,12 @@ func (s *State) DeletePolicy(ctx context.Context, policyID string) error {
 }
 
 func (s *State) CreateProject(_ context.Context, project *storage.Project) (*storage.Project, error) {
+	items := s.projects.Items()
+
+	if len(items) >= v2_constants.MaxProjects+len(storage.DefaultProjectIDs()) {
```

Comment:
Ok so it's really `MaxCustomProjects`, since we have the additional two (maybe just `(unassigned)`) internal projects, and we'll show those on the dropdown as well...